### PR TITLE
fix(web-app): various small bugs

### LINKED ISSUE #964: Fix typo: correct “stuctured” to “structured”
https://next.carbondesignsystem.com/assets

This section
Screen Shot 2022-07-06 at 3 05 49 PM

  |
  
  |
  
  |
 
Screen Shot 2022-07-06 at 3 06 01 PM

### LINKED ISSUE #968: Replace image with higher resolution version
This image is low quality.
https://next.carbondesignsystem.com/developing#understand-the-ecosystem

Current:
Screen Shot 2022-07-06 at 7 12 56 PM

The one used on this page is significantly better.
https://next.carbondesignsystem.com/about-carbon/how-carbon-works#the-carbon-ecosystem

### LINKED ISSUE #974: Web App: SVG libraries top spacing is off
**Description**

Both icons and pictograms pages seem to have a bigger top spacing than necessary, I think `.svg-page` should use
`padding-top: spacing.$spacing-05;` instead of `padding-top: spacing.$spacing-11 + spacing.$spacing-05;`

**Steps to reproduce**

See: http://next.carbondesignsystem.com/elements/pictograms/library vs https://carbondesignsystem.com/guidelines/pictograms/library/
And https://next.carbondesignsystem.com/elements/icons/library vs https://carbondesignsystem.com/guidelines/icons/library/

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #974 
- fix top spacing
http://localhost:3000/elements/icons/library
http://localhost:3000/elements/pictograms/library

Closes #968 
- Update image path to higher res image, delete duplicate
http://localhost:3000/developing

Closes #964 
- fix spelling error
http://localhost:3000/

### CODE REVIEW

File: services/web-app/components/svg-libraries/svg-library.module.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
 .svg-page {
   // prevent jank while importing icons
   min-height: 60vh;
-  padding-top: spacing.$spacing-11 + spacing.$spacing-05;
+  padding-top: spacing.$spacing-05;
```
Comment (francinelucca) [inline services/web-app/components/svg-libraries/svg-library.module.scss:12]: Still see a small padding difference with carbondesignsystem.com, I think it should be spacing-07 here
```suggestion
  padding-top: spacing.$spacing-07;
```

**Review by francinelucca — CHANGES_REQUESTED** [review]
mostly LGTM, small CR

**Review by francinelucca — APPROVED** [review]
💃 🚀

**Review by andreancardona — APPROVED** [review]
yay! looks good

**Pull request merged**